cluster.go

```diff
@@ -37,7 +37,7 @@ func NewClusterFromURL(url, databaseID, token string, timeout time.Duration) (*g
 }
 
 func NewCluster(dialer gocql.HostDialer, username, password string) *gocql.ClusterConfig {
-	cluster := gocql.NewCluster("0.0.0.0") // Placeholder, maybe figure how to make this better
+	cluster := gocql.NewCluster("0.0.0.1") // Placeholder, maybe figure how to make this better
```

Comment:
The [relevant function](https://pkg.go.dev/net/netip#Addr.IsUnspecified) in the golang stdlib (based on [this function](https://github.com/gocql/gocql/blob/master/host_source.go#L160-L162)) looks like it's literally checking only for the IPv4 address we happened to supply (let's leave IPv6 out of it for now).  Just our bad luck I suppose.  Upshot is that literally _anything_ other than 0.0.0.0 should be fine.

With that in mind I think I'd prefer something other than 0.0.0.1 here.  That actually is a public IP which seems to invite confusion: is there a reason we're using that as a default in Astra connection logic?  Does that IP somehow route to Astra?  To avoid all of that I guess I'd rather use either something from one of the private IP address spaces or (alternately) the broadcast address of 255.255.255.255.  The broadcast address is a pretty distant second place to me, though, since it's subject to the same concern ("do they somehow need to broadcast to every node on the network in order to connect to Astra?").  Using a private IP address avoids all of that entirely.

Assuming it doesn't pose an issue with isUnspecified() I guess my vote would be to just use "127.0.0.1" here.

Reply:
The issue with `127.0.0.1` is that it actually is a valid routable ip address while `0.0.0.1` (or any address in the `0.0.0.0/8` range) is reserved and unroutable. But I did consider using `127.0.0.1` here so I can make that change.

Reply:
done

Reply:
127.0.0.0/8 is actually in a pretty similar position to 0.0.0.0/8; it's also a reserved range (although it's meaning is slightly different).  My hope here was that seeing an address so closely associated with loopback would make it clearer that that address wasn't actually used... that may be a false hope. :(